.github/workflows/integration_test.yaml

```diff
@@ -187,6 +201,12 @@ jobs:
         with:
           name: ${{ matrix.image }}
           path: /tmp/${{ matrix.image }}.tar
+      - name: Run Github Trivy Image Action
+        if: ${{ github.event.pull_request.head.repo.fork && inputs.trivy-image-enabled }}
```

Comment:
We don't have this param anymore

Reply:
I miss a Head Banging Against Wall emoji here now...